docs/legal/dmca.md

```
@@ -0,0 +1,27 @@
+If you believe that your copyrighted work has been copied in a way that constitutes copyright infringement and is accessible on this site, please notify our designated copyright agent with the following information:
+
+1. A physical or electronic signature of a person authorized to act on behalf of the owner of the copyright interest.
+2. A description of the copyrighted work that you claim has been infringed.
+3. A description of where the material that you claim is infringing is located on the site, including a URL or other specific location.
+4. Your address, telephone number, and email address.
+5. A statement by you that you have a good faith belief that the disputed use is not authorized by the copyright owner, its agent, or the law.
+6. A statement by you, made under penalty of perjury, that the above information in your notice is accurate and that you are the copyright owner or authorized to act on the copyright owner's behalf.
+
+**Contact Information:**
+
+Email: copyright@yukinu.tiesen.id.vn
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)